### PR TITLE
Booking details modals

### PR DESCRIPTION
**Pull request by mic4ael**

### CODE REVIEW

File: indico/modules/rb/client/js/common/bookings/BookingDetails.jsx [hunk 1]
```diff
@@ -479,9 +479,10 @@ class BookingDetails extends React.Component {
     const showActionButtons = !isCancelled && !isRejected && (canCancel || canReject || showAccept);
     const activeBookings = _.omitBy(occurrences.bookings, value => _.isEmpty(value));
     const occurrenceCount = Object.keys(activeBookings).length;
+
     return (
       <>
-        <Modal onClose={onClose} size="large" closeIcon open>
+        <Modal onClose={() => onClose()} size="large" closeIcon open>
```
Comment (ThiefMaster) [inline indico/modules/rb/client/js/common/bookings/BookingDetails.jsx]: why was this needed? the arguments passed by the modal causing problems?

Comment (mic4ael) [reply indico/modules/rb/client/js/common/bookings/BookingDetails.jsx]: discussed in person
